modules/integration/tests-integration/tests-backend/src/test/resources/artifacts/IS/userMgt/readOnlyLdapUserMgtConfig.toml

```diff
@@ -1,8 +1,7 @@
 [server]
```

Comment:
rename file as mentioned in above comments

Reply:
Fixed.